lib/src/utility/bundle/bundle_decoder.dart

```diff
@@ -0,0 +1,52 @@
+// ignore_for_file: lines_longer_than_80_chars
+
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:vexana/vexana.dart';
+
+class BundleDecoder extends _BaseBundleDecoder with _BundleHelpers {
+  BundleDecoder(super.assetPath);
+
+  Future<R?> crackBundle<T extends INetworkModel<T>, R>({required T model}) async {
+    try {
+      final decodedData = await loadBundle<T, R>(super.assetPath, model: model);
+      return decodedData;
+    } catch (e) {
+      rethrow;
+    }
+  }
+}
+
+mixin _BundleHelpers {
+  Future<R?> loadBundle<T extends INetworkModel<T>, R>(String path, {required T model}) async {
+    try {
+      final bundle = await rootBundle.loadString(path);
+      return await parse<T, R>(bundle, model: model);
```

Comment:
can we do this parse with compute fucntion, if we can, it will be work better performce

Reply:
fixed @VB10